cpp/cmake/Modules/ConfigureCUDA.cmake

```diff
@@ -32,6 +32,9 @@ list(APPEND CUSPATIAL_CUDA_FLAGS -Xcompiler=-Wall,-Werror,-Wno-error=deprecated-
 
 # Produce smallest binary size
 list(APPEND CUSPATIAL_CUDA_FLAGS -Xfatbin=-compress-all)
+if(CMAKE_CUDA_COMPILER_ID STREQUAL "NVIDIA" AND CMAKE_CUDA_COMPILER_VERSION VERSION_EQUAL 12.9)
+  list(APPEND CUSPATIAL_CUDA_FLAGS -Xfatbin=--compress-level=3)
+endif()
```

Comment:
Should it be `VERSION_EQUAL` or `VERSION_GREATER_EQUAL`?

Reply:
This is only required for 12.9, so that is why we are using `VERSION_EQUAL`